vmdk_plugin/utils/test_util/VolumeCreationTestUtil.go

```diff
@@ -49,3 +52,83 @@ func DeleteVolume(name string, ip string) ([]byte, error) {
 func InvokeCommand(remoteHostIP string, cmd string) ([]byte, error) {
 	return exec.Command("/usr/bin/ssh", append(SSH_OPTS, "root@"+remoteHostIP, cmd)...).CombinedOutput()
 }
+
+// returns disk format of volume using Admin cli
+func GetDiskFormatUsingAdminCli(volName string, hostname string) string {
+	cmd := ADMIN_CLI_LS + "-c volume,disk-format 2>/dev/null | grep " + volName
+	op := ExecCmd(hostname, cmd)
+	result := strings.Fields(op)
+	return result[1]
+}
+
+// returns disk format of volume using docker cli
+func GetDiskFormatUsingDockerCli(volName string, hostname string) string {
+	cmd := DOCKER_CLI_INSPC + " --format '{{index .Status \"diskformat\"}}' " + volName
+	op := ExecCmd(hostname, cmd)
+	op = string(op)
+	return strings.TrimSpace(op)
+}
+
+// returns attached to vm field of volume using docker cli
+func GetVmAttachedToVolUsingDockerCli(volName string, hostname string) string {
+	cmd := DOCKER_CLI_INSPC + " --format '{{index .Status \"created by VM\"}}' " + volName
+	op := ExecCmd(hostname, cmd)
+	return strings.TrimSpace(string(op))
+}
+
+// returns attached to vm field of volume using admin cli
+func GetVmAttachedToVolUsingAdminCli(volName string, hostname string) string {
+	cmd := ADMIN_CLI_LS + "-c volume,attached-to 2>/dev/null | grep " + volName
+	op := ExecCmd(hostname, cmd)
+	volProps := strings.Fields(op)
+	return string(volProps[1])
+}
+
+// returns capacity of volume using docker cli
+func GetVolSizeUsingDockerCli(volName string, hostname string) string {
+	cmd := DOCKER_CLI_INSPC + " --format '{{ .Status.capacity.size}}' " + volName
+	op := ExecCmd(hostname, cmd)
+	return strings.TrimSpace(string(op))
+}
+
+// returns capacity of volume using admin cli
+func GetVolSizeByAdminCli(volName string, hostname string) string {
+	cmd := ADMIN_CLI_LS + "-c volume,capacity 2>/dev/null | grep " + volName
+	op := ExecCmd(hostname, cmd)
+	volProp := strings.Fields(op)
+	return volProp[1]
+}
+
+// returns docker version
+func GetDockerVersion(hostname string) string {
+	cmd := "docker -v"
+	return ExecCmd(hostname, cmd)
+}
+
+//method takes command and host and calls InvokeCommand
+//and then returns the output after converting to string
+func ExecCmd(hostname string, cmd string) string {
+	fmt.Println("\n command: ", cmd)
+	io, err := InvokeCommand(hostname, cmd)
+	if err != nil {
+		log.Fatal(err)
+	}
+	return string(io[:])
+}
+
+/*
+docker volume inspect on docker 1.11
```

Comment:
nit: please format this section the same way other comments are fomatted.
Either 
```
 //
 //
```
or 
``` 
/* 
 * xxx
 */
```